bitmind/constants.py

```diff
@@ -27,6 +27,15 @@
     ]
 }
 
+VALIDATOR_DATASET_META = {
```

Comment:
Question: Should we rename DATASET_META into TRAINING_DATASET_META for clarity? Or something more general, considering we also have the face-specific datasets. @dylanuys